mlflow/utils/file_utils.py

```diff
@@ -695,65 +694,31 @@ def parallelized_download_file_using_http_uri(
     """
 
     def run_download(range_start, range_end):
-        template = """
------ stdout -----
-{stdout}
-
------ stderr -----
-{stderr}
-"""
         with tempfile.TemporaryDirectory() as tmpdir:
             json_file = os.path.join(tmpdir, "http_error.json")
-            try:
-                subprocess.run(
-                    [
-                        sys.executable,
-                        download_cloud_file_chunk.__file__,
-                        "--range-start",
-                        str(range_start),
-                        "--range-end",
-                        str(range_end),
-                        "--headers",
-                        json.dumps(headers or {}),
-                        "--download-path",
-                        download_path,
-                        "--http-uri",
-                        http_uri,
-                        "--temp-file",
-                        json_file,
-                    ],
-                    text=True,
-                    check=True,
-                    capture_output=True,
-                    timeout=MLFLOW_DOWNLOAD_CHUNK_TIMEOUT.get(),
-                    env=env,
-                )
-            except TimeoutExpired as e:
-                raise _ChunkDownloadError(
-                    True,
-                    template.format(
-                        stdout=e.stdout.strip() or "(no stdout)",
-                        stderr=e.stderr.strip() or "(no stderr)",
-                    ),
-                ) from e
-            except CalledProcessError as e:
-                retryable = False
-                status_code = None
-                if os.path.exists(json_file):
-                    with open(json_file) as f:
-                        data = json.load(f)
-                        retryable = data.get("retryable", False)
-                        status_code = data.get("status_code")
-                raise _ChunkDownloadError(
-                    retryable,
-                    template.format(
-                        stdout=e.stdout.strip() or "(no stdout)",
-                        stderr=e.stderr.strip() or "(no stderr)",
-                    ),
-                    status_code,
-                ) from e
-            except Exception as e:
-                raise _ChunkDownloadError(False, str(e)) from e
+            subprocess.run(
+                [
+                    sys.executable,
+                    download_cloud_file_chunk.__file__,
+                    "--range-start",
+                    str(range_start),
+                    "--range-end",
+                    str(range_end),
+                    "--headers",
+                    json.dumps(headers or {}),
+                    "--download-path",
+                    download_path,
+                    "--http-uri",
+                    http_uri,
+                    "--temp-file",
+                    json_file,
+                ],
+                text=True,
+                check=True,
```

Comment:
We have `check=True` here. If something goes wrong in the subprocess, an exception should be thrown.